applications/feedback_sentiment_analyzer/SPECIFICATION.md

```diff
@@ -0,0 +1,224 @@
+# FSA Tech spec
+
+This document contains the technical specifications for Feedback Sentiment Analyzer (FSA), a sample application that showcases AWS services and SDKs.
```

Comment:
Can we specify the services and SDKs here?

Reply:
Link back to README.md.